docs/pages/enterprise/sso/adfs.mdx

```diff
@@ -22,6 +22,15 @@ like:
   This guide requires a commercial edition of Teleport.
 </Admonition>
 
+## Prerequisites
+
+- Either an Enterprise version of Teleport (downloaded via the [Enterprise dashboard](https://dashboard.gravitational.com/web/login)) or a Teleport Cloud account (sign up for a [free trial](https://goteleport.com/signup)).
```

Comment:
- Does this mean that the user needs a running Auth/Proxy service? 

- I think we can adjust the visibility of this sentence based on scope, i.e., so Cloud users don't see Enterprise-specific information. (Similar to this section: https://goteleport.com/docs/access-controls/getting-started/#prerequisites)